<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutDown</class>
 <widget class="QMainWindow" name="ShutDown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关机程序</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/shutdown.png</normaloff>:/images/shutdown.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>定时关机</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#55aa00;&quot;&gt;关机时间:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTimeEdit" name="timeEdit_time">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="displayFormat">
              <string>hh:mm:ss</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ff0000;&quot;&gt;剩余时间:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_timer">
             <property name="text">
              <string>启动定时关机</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/shutdown.png</normaloff>:/images/shutdown.png</iconset>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btn_cancel">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>取消定时关机</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_showDown">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>关  机</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/shutdown.png</normaloff>:/images/shutdown.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_reboot">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>重  启</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/reboot.png</normaloff>:/images/reboot.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_exit">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>退出程序</string>
             </property>
             <property name="icon">
              <iconset resource="images.qrc">
               <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>278</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionD">
   <property name="text">
    <string>d</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
